### ISSUE TITLE
Allow deleting one's own comments on another person's website

### ISSUE THREAD

**Issue by dcalhoun** [wordpress-mobile/WordPress-iOS#23770]
We received a feature request from a user (8973860-zd-a8c) to allow deleting/trashing their own comment on someone else's website. 

In a web browser one can "trash" their own comment by clicking _Edit_ → _Trash_, but this ability is absent in the mobile app. Additionally, trashed comments can be only be permanently deleted by the website's administrators. 

Related:
* 8973860-zd-a8c
* https://github.com/wordpress-mobile/WordPress-Android/issues/21423

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by kean** [comment]
It looks like you can't do that on the web either.

**Comment by dcalhoun** [comment]
Yeah, a similar response was shared in https://github.com/wordpress-mobile/WordPress-Android/issues/21423#issuecomment-2457529779. 

I could've sworn I was able to delete my comment on the web, but I'm now unable to reproduce that. I'm guessing I confused myself by using a test account that happened to already be a "member" of the site on which I published a comment. 

Closing this given it is not possible on web either. We can re-open later, if needed.
